packages/babel-plugin-transform-react-constant-elements/README.md

```diff
@@ -1,11 +1,54 @@
 # babel-plugin-transform-react-constant-elements
 
-Treat React JSX elements as value types and hoist them to the highest scope
+Hoists element creation to the top level for subtrees that are fully static, which reduces both allocations and calls to `React.createElement`. More importantly, it tells React that the subtree hasn't changed so React can completely skip it when reconciling.
```

Comment:
This paragraph will be cut out once on the website.

To be consistent, you are missing the description in comment (`>`). [for example](https://github.com/babel/babel/blob/master/packages/babel-preset-es2017/README.md).